pkg/reconciler/common/releases.go

```diff
@@ -213,6 +225,17 @@ func componentDir(instance v1alpha1.KComponent) string {
 	return ""
 }
 
+func additionalManifestPath(version string, instance v1alpha1.KComponent) string {
```

Comment:
```suggestion
func additionalManifestPath(instance v1alpha1.KComponent) string {
```

Is it not alright without version arg? `version := TargetVersion(instance)`  should be called inside the func.
(The same commend for `targetManifestPath`.)

Reply:
done